src/app/ui/AppProvider/NavigationProvider.tsx

```diff
@@ -15,10 +21,33 @@ const theme = {
   },
 };
 
+const linking: LinkingOptions<ReactNavigation.RootParamList> = {
+  prefixes: [DEEP_LINK_PREFIX!],
```

Comment:
I already check if it is `undefined` here:

https://github.com/ChildMindInstitute/mindlogger-app-refactor/blob/a7dac0241ab5d1862ad029b8b3faebd627151709/src/app/ui/AppProvider/NavigationProvider.tsx#L50

So the linking config is only provided if it is not `undefined`. I can update this to make an error log, but I'm not sure what the benefit would be. Do we receive error logs from the mobile app in something like CloudWatch?

Reply:
@sultanofcardio We do collect logs. Right now, we don't use CloudWatch but S3 to store them.
Logs have helped us a lot to investigate bugs that are hard to reproduce or in case we have little information from users. So I think it would be beneficial to add them for deep links too.

Reply:
Makes sense to me. Gonna add an error log in there 👍